.circleci/regenerate.py

```diff
@@ -14,68 +14,78 @@
 https://github.com/pytorch/vision/pull/1321#issuecomment-531033978
 """
 
+import os.path
+
 import jinja2
-from jinja2 import select_autoescape
 import yaml
-import os.path
+from jinja2 import select_autoescape
 
 
 PYTHON_VERSIONS = ["3.6", "3.7", "3.8", "3.9"]
 
 RC_PATTERN = r"/v[0-9]+(\.[0-9]+)*-rc[0-9]+/"
 
 
-def build_workflows(prefix='', filter_branch=None, upload=False, indentation=6, windows_latest_only=False):
+def build_workflows(prefix="", filter_branch=None, upload=False, indentation=6, windows_latest_only=False):
```

Comment:
Is the a way to configure ufmt to avoid making this `' -> "` change?

Reply:
I don't think so, using double quotes is black's convention

Reply:
`black` is the tool ding this. If we don't want it, we could put a `skip_string_normalization = true` line into its section in the `pyproject.toml`.

Reply:
I feel this is adding a number of unnecessary changes, given pep8 doesn't recommend one over the other.

Reply:
Unfortunately this isn't something we can change, as the internal `arc lint` would force these back to double quotes

Reply:
I thought we were not using the default internal linter. Can't we configure that the same way as pointed above by Philip?

Reply:
I don't think it's possible, but even if it were I wouldn't recommend it. It's better to stick to the usual convention as much as possible. The only one we don't properly respect is the line length, for historical reasons. Are you mostly concerned about git blame noise?

Reply:
I have the following concerns here:
1. Unnecessary changes.
2. I am not convinced with the idea of forcing `"` over '`'. This should be a user choice given that pep8 doesn't recommend either. Also, this is not the usual convention. Line length is something recommend by pep8.
3. Python by default uses `'` for representing string:
```
>>> a = "abc"
>>> a
'abc'
```
4. `'` uses 1 key stroke against 2 for `"`.

Reply:
regarding 4, you're free to use single quotes if you want and let `black` automatically format them :)

For the rest, I understand your point, but one of the goal of using `black` is precisely to avoid that kind of discussion about  style preferences. It may not be perfect and suit everyone's taste (it doesn't suit mine either), but at least it's a standard that we can all automatically comply to.